packages/dd-trace/src/profiling/exporters/agent.js

```diff
@@ -83,7 +89,28 @@ class AgentExporter {
         `profiler_version:${version}`,
         'format:pprof',
         ...Object.entries(tags).map(([key, value]) => `${key}:${value}`)
-      ].join(',')
+      ].join(','),
+      info: {
```

Comment:
No particular reason; I gave it some thought while writing the code, but figured that since `event` is constructed once per minute, this is not performance critical and we might as well create the whole `event` object as a transient value and not have parts of it take up memory when not used. It's all then expressed in one place in the source code as well. It still needs to be serialized to JSON.

Reply:
Ok, we can always change that in the future if we need to collect information that is more expensive to compute.